ipa-core/src/ff/galois_field.rs

```diff
@@ -153,6 +156,15 @@ macro_rules! bit_array_impl {
                 }
             }
 
+            impl $name {
+
+                #[inline]
+                #[must_use]
+                pub fn as_raw_slice(&self) -> &[u8] {
```

Comment:
@andyleiserson do you have any concerns about implicit endianness here?

Reply:
I actually do not need this, so I will remove it.